compute/admin_guide/alerts/email.adoc

```diff
@@ -51,23 +51,56 @@ include::frag_config_triggers.adoc[leveloffset=0]
 . In *Credential*, create the credentials required to access the email account that sends alerts.
 This isn't a required field.
 
-.. Click *Add new*.
+.. Select *Add new*.
 
 .. Select *Basic authentication*.
 
 .. Enter a username and password.
 
-. If you're using SMTPS (your SMTP connection is secured by SSL), set *SSL* to *On*.
+. If you are using SMTPS, where the SMTP connection is encrypted when estabilshed, set *SSL* to *On*.
++
+NOTE: To support email alerts for SMTP connections encrypted using the `STARTTLS` command, disable *SSL*.
 
 . Set up your recipients.
 
-.. Click *Add recipient*, and enter an email address.
+.. Select *Add recipient*, and enter an email address.
 Every email alert profile must have at least one recipient, even if you're using alert labels.
 
 .. (Optional) Specify recipients using xref:../audit/annotate_audits.adoc[alert labels].
 
-. Click *Next*.
+. Select *Next*.
 
 . Review the *Summary* and test the configuration by selecting *Send test alert*.
 
-. Click *Save*.
+. Select *Save*.
+
+=== Troubleshooting
+
+[.task]
+
+==== Unable to test Email Alerts for 'smtp.office365.com' on port 587.
```

Comment:
@divyabhushan can you please confirm if this topic is in PCCE and PCEE AG?

If yes, is this section for PCCE only?

Reply:
@Pubs-MV Yes, this topic and section is for both PCCE and PCEE. Thanks

Reply:
Can you please confirm this with QA once.